CONTRIBUTING.md

```diff
@@ -42,10 +87,6 @@ We recommend you to use our prebuilt Docker image to start your development work
   python setup.py develop
   ```
 
-### Build PyTorch/XLA from source with GPU support
-
-Please refer to this [guide](https://github.com/pytorch/xla/blob/master/docs/gpu.md#develop-pytorchxla-on-a-gpu-instance-build-pytorchxla-from-source-with-gpu-support).
```

Comment:
this is actually useful, user has to run
```
echo "export PATH=\$PATH:/usr/local/cuda-12.1/bin" >> ~/.bashrc
echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:/usr/local/cuda-12.1/lib64" >> ~/.bashrc
source ~/.bashrc
```
to build on GPU docker.

Reply:
I've needed to do that before inside containers in GKE (which uses a different container runtime), but never with Docker with `nvidia-container-toolkit` installed. Are you sure that's still necessary?

Reply:
Last time I tried(~1 month ago) I still need it with the GPU docker.

Reply:
Got it. I'll revert this change for now and give it an update next time I break my GPU build machine